apex/python/log.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import time
+
+class message:
+    def __init__(self):
+        self.start_time = time.time()
+        logging.basicConfig(filename="apex_tacos.log", filemode="w", format="%(name)s - %(levelname)s - %(message)s", level=logging.INFO)
+        logging.info("Starting Logger..")
+
+    def display_info(self, s:str) ->None:
```

Comment:
maybe use a more obvious method name for a logger to log at different levels. For example: log_info, log_warning etc